Plan.js

```diff
@@ -10,7 +10,9 @@ class Plan {
     course_bank = [course, course, ...]
   */
   constructor(major, start_season, start_year){
-    this.major = MAJORS[0]; //TEMP FIX. this.major was pulling the major name only instead of the major object. This sets this.major = the first major object in the array of majors.
+    const value = document.getElementById('majorSelect').value+"";
```

Comment:
Instead of reading elements from the page here, you should leave that to Executive (like how it determines the start season/year) and pass the string "Computer Science" as the first parameter of this constructor.